float32 x_center
float32 y_center
float32 size_x
float32 size_y
